tracer/test/Datadog.Trace.IntegrationTests/FleetInstaller/RegistryHelperTests.cs

```diff
@@ -0,0 +1,188 @@
+﻿// <copyright file="RegistryHelperTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#if NETFRAMEWORK
+
+using System;
+using Datadog.FleetInstaller;
+using FluentAssertions;
+using Microsoft.Win32;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.IntegrationTests.FleetInstaller;
+
+public class RegistryHelperTests(ITestOutputHelper output) : FleetInstallerTestsBase(output)
+{
+    [SkippableFact]
+    public void AddCrashTrackingKey_AddsKeyIfItDoesntExist()
```

Comment:
What about cleanup? I know we don't really care in the CI, but this is going to make a mess when running locally

Reply:
We're cleaning up the monitoring home values, so it's "just" the crashtracking registry keys we're leaving lying around, but yeah, probably should.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/6714/commits/47036dde76dc3cec7114e34ddb21472004d21a81